content/yaml-publishing/firebase-app-distribution.md

```diff
@@ -180,28 +151,25 @@ buildTypes {
         ...
         firebaseAppDistribution {
             ...
-            serviceCredentialsFile="<your/file/path.json>"
+            serviceCredentialsFile=System.getenv('GOOGLE_APPLICATION_CREDENTIALS')
         }
     }
 {{< /highlight >}}
 
 
-Note that in case the credentials file is not specified in `firebaseAppDistribution` build type, the environment variable `GOOGLE_APPLICATION_CREDENTIALS` will be used instead.
-
-<br>{{<markdown>}}
-9. Decode application credentials for Firebase authorization:{{</markdown>}}
+2. Decode application credentials for Firebase authorization:
 
 {{< highlight yaml "style=paraiso-dark">}}
 scripts:
   -name: Decode Google credentials
    script: | 
-     echo $GOOGLE_APP_CREDENTIALS > $CM_BUILD_DIR/your/file/path.json
+     echo $FIREBASE_SERVICE_ACCOUNT > $GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
Might want to consider changing the variable name to something that reflects it's a file path e.g. `GOOGLE_APPLICATION_CREDENTIALS_PATH` in case people start adding the JSON value to it by mistake?

Reply:
Keep it like that then to be consistent.